recipes/tip/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "tip" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b1db8a04d2cfad8443e02f96e80703777dd99412d0d6772d7a59221f67cb3470
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make 4.*  # [unix]
+    - ninja 1.*
+    - cmake 3.*
+    - pkgconfig 1.*
+
+  host:
+    - arrow-cpp 6.*
+    - spdlog 1.* 
+    - yaml-cpp 0.*
+    - gmock 1.*
+    - gtest 1.* 
+    - libtins 4.*
+    - fmt 9.*
+
+  run:
+    - {{ pin_compatible('ucrt', max_pin='x') }}  # [win]
```

Comment:
```suggestion
```
Do no manually pin UCRT, this package is exported/controlled by the compiler package.

Reply:
If this is not present, I get overlinking errors.

Reply:
@conda-forge/staged-recipes

Reply:
Please provide a link to the overlinking error in the logs that you are talking about.

Reply:
Same story as above. It does pass in the CI as confirmed with the most recent run. Previously I didn't bother pushing unless it built locally. 

Error output after removing line 34 on my local windows machine: 
```
ERROR (tip,Library/bin/pqcompare.exe): .. but ['ucrt'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

Reply:
Unlike for Linux, our Windows build environment is not trivially replicable on your local machine. I assume you got this error because you aren't using the package pinnings from conda-forge-pinnings. So your build isn't getting our compiler packages which have the appropriate run exports.

Reply:
In order to use our pins for a local windows build, you would need to copy [this yaml](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml) into the recipe directory.